frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/vm-settings-tab-state-update.ts

```diff
@@ -103,6 +108,28 @@ export const flavorUpdater = ({ id, prevState, dispatch, getState }: UpdateOptio
   );
 };
 
+export const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
+  const state = getState();
+  if (!hasVMSettingsValueChanged(prevState, state, id, VMSettingsField.OPERATING_SYSTEM)) {
+    return;
+  }
+  const os = iGetVmSettingValue(state, id, VMSettingsField.OPERATING_SYSTEM);
+  const isWindows = os && os.startsWith('win');
```

Comment:
@mareklibra @suomiy is there a better way to detect if an os is Windows?

Reply:
I think it should be good enough as all windows ids start with "win". And it seems there is no other exact information (like family) in the template